src/index.html

```diff
@@ -8,6 +8,7 @@
 </head>
 <body>
   <h1 class="header">Todo List</h1>
+  
   <ul class="todos"></ul>
 </body>
```

Comment:
### Good work creating the differents functions for the CRUD actions. Create, Read, Update and Delete. 💯 💯 💯 💯 

- [ ] Kindly add in your body the corresponding `input` where the user can create a task 🟢 
- [ ] Kindly add the corresponding event listener to this `input` so the user can add a task 🟢 
- [ ] Kindly add also a checkbox for each task, so the index of this one will be the one that's deleted or created, and the Local storage it's updated with these changes 🟢 
- [ ] Kindly add the delete icon to delete a checked task 🟢 
- [ ] Kindly add the delete all button to delete all the tasks that are selected 🟢 

![image](https://user-images.githubusercontent.com/93630700/207458367-18806fdd-c574-4fd5-b6e8-b71932de91b3.png)